src/library/searchqueryparser.h

```diff
@@ -27,6 +32,8 @@ class SearchQueryParser {
     QString getTextArgument(QString argument,
                             QStringList* tokens) const;
 
+    TrackCollection* m_pTrackCollection;
+
     QSqlDatabase m_database;
```

Comment:
Also related to the previous comment. If I get rid of this, then the test class can't work
because it uses the constructor that attaches a connection to m_database